scss/_alert.scss

```diff
@@ -119,26 +133,17 @@
   --#{$prefix}alert-padding-x: 0;
   --#{$prefix}alert-padding-y: #{$alert-padding-sm};
   --#{$prefix}alert-margin-bottom: 0;
+  @include rfs($font-size-sm, --#{$prefix}alert-font-size);
+  --#{$prefix}alert-line-height: #{$line-height-sm};
   --#{$prefix}alert-border: 0;
+  --#{$prefix}alert-logo-size: #{$alert-logo-size-sm};
+  --#{$prefix}alert-icon-size: #{divide($alert-icon-size, 2)};
+  --#{$prefix}alert-btn-close-offset: #{$alert-btn-close-offset-sm};
   // scss-docs-end alert-sm-css-vars
-
-  font-size: $font-size-sm;
-  line-height: $line-height-sm;
+  --#{$boosted-prefix}icon-spacing: #{$btn-close-padding-sm};
```

Comment:
Can we have button close with small alerts? If not, I'm not sure this CSS var is useful.

Reply:
IMO, this should be tackled inside another PR. That said, I think that every alerts should be dismissible (even if we don't show it on our doc).

Reply:
OK you can let this modification in the PR but let's create an issue to track it